tests/autoyast/prepare_profile.pm

```diff
@@ -31,6 +32,24 @@ sub run {
     # Get file from data directory
     my $profile = get_test_data($path);
 
+    # Add registration block for cloned autoyast profiles due to
+    # the registration code is never clonned as it is not stored in the system anywhere
+    if (get_var('AUTOYAST_ADD_REGISTRATION')) {
+        # Create registration block
+        my $registration = << "EOF";
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <email/>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <install_updates config:type="boolean">true</install_updates>
+    <reg_server>{{SCC_URL}}</reg_server>
+  </suse_register>
+  <add-on>
+EOF
+        # Inject registration block
+        $profile =~ s/<add-on>/$registration/;
```

Comment:
from the UX point of you could add a record_info as the http://rivera-workstation.suse.cz/tests/782#step/prepare_profile/1 doesnt provide any visual info.

Reply:
might help, it is just is running in the worker directly so, for example message are not displayed if the the test suite fails before some UI test, so normally I don't pay much attention to UI in this modules, I think it should be in the logs in these cases. But thanks, added anyway.